lib/guides_style_18f/navigation.rb

```diff
@@ -101,10 +154,14 @@ def self.nav_data_by_title(nav_data)
   private_class_method :nav_data_by_title
 
   def self.page_nav(front_matter)
-    { 'text' => front_matter['title'],
-      'url' => "#{front_matter['permalink'].split('/').last}/",
+    url_components = front_matter['permalink'].split('/')[1..-1]
+    result = {
+      'text' => front_matter['title'],
+      'url' => "#{url_components.nil? ? '' : url_components.last}/",
       'internal' => true,
     }
+    result.delete 'url' if result['url'] == '/'
```

Comment:
Why does the URL need to be deleted if it's at the root?  Is this related to the issue you were explaining yesterday?